api/converter/from_pb.go

```diff
@@ -665,8 +670,12 @@ func FromUpdatableProjectFields(pbProjectFields *api.UpdatableProjectFields) (*t
 	if pbProjectFields.AuthWebhookMethods != nil {
 		updatableProjectFields.AuthWebhookMethods = &pbProjectFields.AuthWebhookMethods.Methods
 	}
-	if pbProjectFields.ClientDeactivateThreshold != 0 {
-		updatableProjectFields.ClientDeactivateThreshold = gotime.Duration(pbProjectFields.ClientDeactivateThreshold)
+	if pbProjectFields.ClientDeactivateThreshold != nil {
+		clientDeactivateThreshold, err := gotime.ParseDuration(pbProjectFields.ClientDeactivateThreshold.Value)
+		if err != nil {
+			updatableProjectFields.ClientDeactivateThreshold = 24 * gotime.Hour
```

Comment:
It would be better to check if `ClientDeactivateThreshold` requested by the Client is valid and return a proper error if not.

https://github.com/yorkie-team/yorkie/blob/c5b9a908577b911c1598496a6f0e83c94563cbe2/server/grpchelper/status.go#L40-L47

Reply:
1. Added `flagClientDeactivateThreshold` in clientCLI's ProjectUpdate.
2. Revised error handling in `time.ParseDuration()` to pass proper error.